modules/extension/xsd/xsd-wfs/src/test/resources/org/geotools/wfs/bindings/test-data/QueryTypeBindingTest.xml

```diff
@@ -18,9 +18,7 @@
 	<wfs:PropertyName>property2</wfs:PropertyName>
 	<wfs:XlinkPropertyName
 		traverseXlinkDepth="*"
-		traverseXlinkExpiry="10">
-		gt:propertyA/gt:propertyB
-	</wfs:XlinkPropertyName>
+		traverseXlinkExpiry="10">gt:propertyA/gt:propertyB</wfs:XlinkPropertyName>
```

Comment:
Same here, no need to change, the trimming here should happen

Reply:
I've put the trimming back in - I'm not sure it is 100% correct but it is the expected behaviour.

Reply:
It's another xs:string, so the comment above is still valid. Might as well be that it was wrong before, let's hear from @bencaradocdavies